e2e/testenv/env.go

```diff
@@ -57,8 +57,10 @@ func (e *Env) getShellEnv() {
 // NewNamespaceName returns a name for a namespace that is unique to the test
 // run. e.g. as a targetNamespace for workloads
 func NewNamespaceName(name string) string {
-	rand.Seed(time.Now().UnixNano())
```

Comment:
What do you think about passing in the randomness source, like you do with `RandomFilename`, and using `GinkgoRandomSeed()` when calling from ginkgo tests?